app/controllers/about_controller.rb

```diff
@@ -5,5 +5,12 @@ def index
     @smart_proxies = SmartProxy.authorized(:view_smart_proxies).includes(:features)
     @compute_resources = ComputeResource.authorized(:view_compute_resources)
     @plugins = Foreman::Plugin.all
+    @providers = ComputeResource.supported_providers.merge(ComputeResource.registered_providers).keys.map do |provider|
```

Comment:
Also, would you mind renaming the `provider` to `provider_name` in the local arg, as that's what it represents (I would first thing it's the actual class)